src/skin/imgloader.cpp

```diff
@@ -25,6 +26,9 @@ QImage* ImgLoader::getImage(const QString& fileName, double scaleFactor) const {
         }
     }
     if (scaleFactor > 1.0) {
+#else
+    if (qApp->devicePixelRatio() > 1.0) {
```

Comment:
Why would the user want to override the system scaling setting? If the system scaling setting does not work well for them, why wouldn't they want to change it for all programs, not just Mixxx?